source/navier_stokes_matrix.cc

```diff
@@ -911,7 +925,10 @@ NavierStokesMatrix<dim>::local_divergence(
                  -mu_values[q]) :
               make_vectorized_array(-1.);
 
-          pressure.submit_value(weight * velocity.get_divergence(q), q);
+          if constexpr (dim == 1)
+            pressure.submit_value(weight * velocity.get_gradient(q), q);
+          else
+            pressure.submit_value(weight * velocity.get_divergence(q), q);
```

Comment:
Same here, I would be fine with adding a `get_divergence` function in the 1D `FEEvaluationAccess` class.